analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/ProjectBuilder.cs

```diff
@@ -99,6 +100,11 @@ public ProjectBuilder AddSnippet(string code, string fileName = null)
         public static ProjectBuilder FromProject(Project project) =>
             new(project);
 
+        private bool IsExtensionMatch(FileInfo fileInfo) =>
```

Comment:
NITPICK, feel free to disregard:

```suggestion
        private bool IsExtensionOfSupportedType(FileInfo fileInfo) =>
```